Core.ApplicationServices/Rights/UserRightsService.cs

```diff
@@ -170,6 +170,29 @@ public Maybe<OperationError> TransferRights(int fromUserId, int toUserId, int or
             );
         }
 
+        public Maybe<OperationError> CopyRights(int fromUserId, int toUserId, int organizationId, UserRightsChangeParameters parameters)
+        {
+            if (fromUserId == toUserId)
+            {
+                return Maybe<OperationError>.None;
```

Comment:
Do we want to be more explicit about what's wrong in this case?

Reply:
It doesn't seem like at the moment the frontend is handling backend messages, so I think sticking to what was already there is fine (this line is already used for the transfer logic)